SimpleCalculator.py

```diff
@@ -90,12 +90,17 @@ def calculate_average(values):
 # TODO this function should take list of values,
 # then returns sorted list in ascending(normal) or descending(reverse" order
 def sort_values(values, order="normal"):
-    for x in range(0,5):
-        for i in range(0,5):
+    for x in range(len(values)):
```

Comment:
Ok, I see that you got little confused :D One thing is you should do make to different sorting, and make some logical check(for example some 'if') and dependng on this check do correct sort. If this is still not clear for you then I can show it by facebook or Skype or whatever, it's nothing to worry about everybody started from nowhere and got confused :D